concepts/recursion/introduction.md

```diff
@@ -0,0 +1,35 @@
+# Introduction
+
+Recursion is a way to repeat code in a function by the function calling itself.
+It can be viewed as another way to loop/iterate. 
+Like looping, a Boolean expression or `True/False` test is used to know when to stop the recursive execution.
+_Unlike_ looping, recursion without termination in Python cannot not run infinitely.
+Values used in each function call are placed in their own frame on the Python interpreter stack.
+If the total amount of function calls takes up more space than the stack has room for, it will result in an error.
+
+```python
+def print_increment(step, max_value):
+    if step > max_value:
+        return
+    print(f'The step is {step}')
+    print_increment(step + 1, max_value)
+
+
+def main():
+    print_increment(1, 2)
+    print("After recursion")
+
+if __name__ == "__main__":
+    main()
+
+```
+
+This will print
+
+```
+The step is 1
+The step is 2
+After recursion
+```
+
+There may be some situations that are more readable and/or easier to reason through when expressed through recursion than when expressed through looping.
```

Comment:
_Below this paragraph is where I'd add in sections around Problems "best" suited to recursion and problems less well suited.  We may then want to put in a "read more" or "get started" section with links (like the one you had originally) with 3-4 links to learn more.  OR we can put all the links into the `links.json` docs -- they'll display at the bottom of the concept doc on the site.  Which you do depends on how much framing you want to put around it.  If you want to introduce them a bit, I'd do in-doc links.  Otherwise, `links.json`._

Reply:
And here we may also want to allude to mitigation strategies.  Or we could simply add in articles that go over those to `links.json`.